contrib/utils/init_embedding.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
```

Comment:
I am fine with having this in `contrib`, but I think this could also be part of the Sockeye package directly. In that case you may be able to reuse the save_params_to_file method in `utils.py`, the load/save vocab methods, and logging setup for example.  Your call :)